<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileSourcePage</class>
 <widget class="QWidget" name="FileSourcePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>70</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="1" column="1">
    <widget class="QComboBox" name="formatCombo">
     <item>
      <property name="text">
       <string>Autodetect</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Raw complex 32-bit float (CF32)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Raw complex 8-bit unsigned (CU8)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Raw complex 8-bit signed (CS8)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Raw complex 16-bit signed (CS16)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>WAV file</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SigMF recording</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>File</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="browseButton">
     <property name="text">
      <string>&amp;Browse...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Format</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="pathEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="loopCheck">
     <property name="text">
      <string>&amp;Loop</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
